lua/astrocommunity/remote-development/remote-sshfs/init.lua

```diff
@@ -0,0 +1,47 @@
+---@type LazySpec
+return {
+	"nosduco/remote-sshfs.nvim",
```

Comment:
```suggestion
  "nosduco/remote-sshfs.nvim",
  cmd = {
    "RemoteSSHFSConnect",
    "RemoteSSHFSDisconnect",
    "RemoteSSHFSEdit",
    "RemoteSSHFSFindFiles",
    "RemoteSSHFSLiveGrep",
  },
```

add lazy loading on command